doc/source/io.rst

```diff
@@ -4562,6 +4562,20 @@ destination DataFrame as well as a preferred column order as follows:
                              index_col='index_column_name',
                              col_order=['col1', 'col2', 'col3'], projectid)
 
+
+You can specify the query configuration as parameter
+
+.. code-block:: python
+
+   config = {
+      'query': {
+        "useQueryCache": False
+      }
+   }
+   data_frame = pd.read_gbq('SELECT * FROM test_dataset.test_table',
+                             configuration=config, projectid)
```

Comment:
Although `config` is shorter, it is 'configuration' that is used by gbq (https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration), so shouldn't we follow gbq's terminology here?